File: SDKs/stackexchange.com/2.0/python/src/sdk/models/operations/get_me_reputation_history_full.py
from dataclasses import dataclass, field
from typing import Optional


@dataclass
class GetMeReputationHistoryFullQueryParams:
    site: str = field(metadata={'query_param': { 'field_name': 'site', 'style': 'form', 'explode': True }})
    callback: Optional[str] = field(default=None, metadata={'query_param': { 'field_name': 'callback', 'style': 'form', 'explode': True }})
    filter: Optional[str] = field(default=None, metadata={'query_param': { 'field_name': 'filter', 'style': 'form', 'explode': True }})
    page: Optional[int] = field(default=None, metadata={'query_param': { 'field_name': 'page', 'style': 'form', 'explode': True }})
    pagesize: Optional[int] = field(default=None, metadata={'query_param': { 'field_name': 'pagesize', 'style': 'form', 'explode': True }})
    

@dataclass
class GetMeReputationHistoryFullRequest:
    query_params: GetMeReputationHistoryFullQueryParams = field()
    

@dataclass
class GetMeReputationHistoryFullResponse:
    content_type: str = field()
    status_code: int = field()
    body: Optional[bytes] = field(default=None)
    

File: SDKs/stackexchange.com/2.0/go/pkg/models/operations/getmereputationhistoryfull.go
package operations



type GetMeReputationHistoryFullQueryParams struct {
    Callback *string `queryParam:"style=form,explode=true,name=callback"`
    Filter *string `queryParam:"style=form,explode=true,name=filter"`
    Page *int64 `queryParam:"style=form,explode=true,name=page"`
    Pagesize *int64 `queryParam:"style=form,explode=true,name=pagesize"`
    Site string `queryParam:"style=form,explode=true,name=site"`
    
}

type GetMeReputationHistoryFullRequest struct {
    QueryParams GetMeReputationHistoryFullQueryParams 
    
}

type GetMeReputationHistoryFullResponse struct {
    Body []byte 
    ContentType string 
    StatusCode int64 
    
}

